src/content/docs/more-integrations/dbt-cloud/dbt-cloud-integration.mdx

```diff
@@ -0,0 +1,536 @@
+---
+title: Dbt Cloud with Airflow and Snowflake Integration 
+tags:
+    - New Relic integrations
+    - Dbt Cloud integrations 
+redirects: 
+  - /docs/install/dbt-cloud-integration
+metaDescription: Dbt Cloud with Airflow and Snowflake integration 
+freshnessValidatedDate: never
+---
+
+Our <DoNotTranslate>dbt Cloud</DoNotTranslate> integration with <DoNotTranslate>Airflow</DoNotTranslate> monitors the health of your <DoNotTranslate>dbt Cloud</DoNotTranslate> jobs and resources, helping you identify problems like when runs, models, or tests fail. 
+
+This integration runs on <DoNotTranslate>Apache Airflow</DoNotTranslate> and queries <DoNotTranslate>Snowflake</DoNotTranslate> for any failed tests if configured to do so.
+
+## Prerequisites [#prerequisites]
+
+* dbt Cloud account with APIs enabled and using <DoNotTranslate>Snowflake</DoNotTranslate> as the database. 
+* Access to the <DoNotTranslate>Snowflake</DoNotTranslate> account where the <DoNotTranslate>dbt Cloud</DoNotTranslate> account runs.
+* Existing <DoNotTranslate>Airflow</DoNotTranslate> environment version 2.8.1 or higher, or or ability to run <DoNotTranslate>Docker Compose</DoNotTranslate>.
+
+## Install the integration [#install]
+
+You can install the New Relic <DoNotTranslate>dbt Cloud</DoNotTranslate> integration with <DoNotTranslate>Airflow</DoNotTranslate> by either:
+
+* Installing in your existing Airflow environment. This is recommended for production environments.
+* Installing with Docker Compose. This is suitable for quick POCs.
+
+Select the option most suitable for your needs by clicking on its tab:
+
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="1">
+            Install on existing Airflow environment
+        </TabsBarItem>
+        <TabsBarItem id="2">
+            Install with Docker Compose
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="1">
+<Steps>
+<Step>
+Ensure you have the Snowflake provider and then clone the `newrelic-dbt-cloud-integration` repository by running these commands:
+
+```shell
+pip install apache-airflow-providers-snowflake>=3.0.0
+```
+
+```shell
+git clone https://github.com/newrelic-experimental/newrelic-dbt-cloud-integration.git
+```
+
+</Step>
+<Step>
+Copy the contents of `airflow/dags` to the root of your Airflow dags folder 
+</Step>
+<Step>
+Create the five Airflow connections needed for the DAG. The following table provides the connection name the info to set it up:
+
+<table>
```

Comment:
While a table makes sense in theory, this feels like too much text. Especially with the links that are broken up into multiple lines. This is difficult to read on a laptop, and will be more difficult on mobile.